File: PaymentSDK/UIHostObjCApp/models/Amount.m
//
//  Amount.m
//  UIHostObjCApp
//
//  Created by Rahul Dhuri on 04/06/19.
//  Copyright © 2019 Network International. All rights reserved.
//

#import "Amount.h"

@implementation Amount

+ (instancetype)initWithCurrency:(NSString *)code andValue:(int)value {
    Amount *amount = [[Amount alloc] init];
    amount.currencyCode = code;
    amount.value = value;
    return amount;
}




@end

File: PaymentSDK/UIHostApp/models/Amount.swift
//
//  Amount.swift
//  merchant-sample-app
//
//  Created by Niraj Chauhan on 4/26/19.
//  Copyright © 2019 Niraj Chauhan. All rights reserved.
//

import Foundation

struct Amount : Codable {
    var currencyCode: String
    var value: Int
}
